### ISSUE TITLE
Change default page offset to 1

### ISSUE THREAD

**Issue by opal06** [lucasrla/remarks#60]
I think that while 0 seems the logical default offset from a programmer's perspective, 1 would actually be a better default as it would fit the average document pagination better. I can make a pull request, but as it's not a big change I wanted to bring it up here first.

Thanks for all the great work, sorry to bother you again!

**Comment by lucasrla** [comment]
You're right. It pains me a bit to change a flag's default value (due to backwards compatibility) but having it as `1` is just better long term.

Happy to merge your PR.

Thanks!

**Comment by opal06** [comment]
@lucasrla I see the argument for backwards compatibility, but if someone really depends on the offset being 0, they should still be able to get that behaviour by setting the flag to 0 on program execution, right?
